### PR TITLE
Feat: added support for rego files to filter input messages

### PR DESCRIPTION
**Pull request by DmitriyLewen**
Issue #175

### CODE REVIEW

File: ui/frontend/src/components/RouteDetails.vue [hunk 1]
```diff
@@ -24,6 +24,17 @@
             :inputHandler="updateField"
             :validator="v(uniqueName)"
           />
+          <PropertyField
```
Comment (AndreyLevchenko) [inline ui/frontend/src/components/RouteDetails.vue:27]: please see
https://github.com/aquasecurity/postee/blob/main/ui/frontend/src/components/OutputDetails.vue#L159-L163
no need to define special handlers here

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

**Comment by AndreyLevchenko** [comment]
@DmitriyLewen 
Coverage of patch isn't very well. It needs to be improved

File: regoservice/regocheck.go [hunk 2]
```diff
@@ -3,31 +3,64 @@ package regoservice
...
+	return filesWithPath
+}
 
-	r := rego.New(
+func buildRegoLoader(files []string, rule string) func(r *rego.Rego) {
+	if len(files) != 0 && files[0] != "" {
+		filesWithPath := getFilesWithPathToRegoFilters(files)
+		return rego.Load(filesWithPath, nil)
+	}
+	if rule == "" { //no rule defined - any input allowed
```
[the remark this answers is not in the document]
Comment (DmitriyLewen) [reply regoservice/regocheck.go]: Done

File: ui/frontend/src/components/RouteDetails.vue (hunk 1 above)
Comment (DmitriyLewen) [reply ui/frontend/src/components/RouteDetails.vue:27]: changed it

File: rego-filters/Policy-Related-Features.rego [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package postee
+import future.keywords.in
```
Comment (AndreyLevchenko) [inline rego-filters/Policy-Related-Features.rego:2]: this causes an error while evaluation

File: rego-filters/Policy-Min-Vulnerability.rego [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package postee
+import future.keywords.in
```
Comment (AndreyLevchenko) [inline rego-filters/Policy-Min-Vulnerability.rego:2]: this causes an error while evaluation

File: rego-filters/Policy-Registry.rego [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package postee
```
Comment (AndreyLevchenko) [inline rego-filters/Policy-Registry.rego:1]: should be named 'Allow-Registry'

File: rego-filters/Policy-Image-Name.rego [hunk 6]
```diff
@@ -0,0 +1,13 @@
+package postee.aa
```
Comment (AndreyLevchenko) [inline rego-filters/Policy-Image-Name.rego:1]: should be named `Allow-Image-Name`

File: msgservice/msghandling.go [hunk 7]
```diff
@@ -27,11 +27,19 @@ func (scan *MsgService) MsgHandling(input []byte, output outputs.Output, route *
...
+	if ok, err := regoservice.DoesMatchRegoCriteria(in, route.InputFiles, route.Input); err != nil {
+		if regoservice.IsUsedRegoFiles(route.InputFiles) {
+			prnInputLogs("Error while evaluating rego rule %s :%v for the input %s", route.Input, err, input)
+		} else {
+			prnInputLogs("Error while evaluating rego rule for input files :%v for the input %s", err, input)
+		}
 		return
 	} else if !ok {
-		prnInputLogs("Input %s... doesn't match a REGO rule: %s", input, route.Input)
+		if regoservice.IsUsedRegoFiles(route.InputFiles) {
```
Comment (AndreyLevchenko) [inline msgservice/msghandling.go:38]:
```
if !regoservice.IsUsedRegoFiles(route.InputFiles)
```

File: rego-filters/Policy-Image-Name.rego (hunk 6 above)
Comment (AndreyLevchenko) [reply rego-filters/Policy-Image-Name.rego:1]: also why is such `package`? - it breaks correct work of rule.
And as `postee` package is required then it should be documented somewhere

File: rego-filters/Igrore-Image-Name.rego [hunk 8]
```diff
@@ -0,0 +1,13 @@
+package postee
```
Comment (AndreyLevchenko) [inline rego-filters/Igrore-Image-Name.rego:1]: should be named Ig**n**ored-Image-Name

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: rego-filters/Igrore-Image-Name.rego (hunk 8 above)
Comment (DmitriyLewen) [reply rego-filters/Igrore-Image-Name.rego]: name changed

File: msgservice/msghandling.go (hunk 7 above)
Comment (DmitriyLewen) [reply msgservice/msghandling.go]: done

File: rego-filters/Policy-Image-Name.rego (hunk 6 above)
Comment (DmitriyLewen) [reply rego-filters/Policy-Image-Name.rego]: name changed
package info added in readme

File: rego-filters/Policy-Registry.rego (hunk 5 above)
Comment (DmitriyLewen) [reply rego-filters/Policy-Registry.rego]: name changed

File: rego-filters/Policy-Min-Vulnerability.rego (hunk 4 above)
Comment (DmitriyLewen) [reply rego-filters/Policy-Min-Vulnerability.rego:2]: OPA package updated

File: rego-filters/Policy-Related-Features.rego (hunk 3 above)
Comment (DmitriyLewen) [reply rego-filters/Policy-Related-Features.rego:2]: OPA package updated

**Review by AndreyLevchenko — APPROVED** [review]

**Pull request merged**